index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!-- NAVBAR HERE -->
+    <nav>
+      <ul>
+        <li class="logo-left">
+          <img src="img/03.png" alt="" srcset="">
+          <p>Original Trombones</p>
+        </li>
+        <li>Feature</li>
+        <li>How it works</li>
+        <li>Pricing</li>
+      </ul>
+    </nav>
+
+    <!-- Container Web Page -->
+
+    <div class="webapp-container">
+        <!-- EMAIL FORM -->
+
+      <div class="email-form">
+        <h2>Handcrafted, home made masterpieces</h2>
+        <input type="email" name="" placeholder="Enter your email address">
+        <button type="button">GET STARTED</button>
```

Comment:
- I think it's better if you include hover, active, and focus on your buttons. and also to change the cursor to a pointer when you are on the button. 
```suggestion
        <button type="button">GET STARTED</button>
```